chrome/content/zotero/xpcom/itemTreeView.js

```diff
@@ -260,6 +261,44 @@ Zotero.ItemTreeView.prototype.setTree = Zotero.Promise.coroutine(function* (tree
 });
 
 
+Zotero.ItemTreeView.prototype.setSortColumn = function() {
+	var colId, col, dir, currentCol, currentDir;
```

Comment:
>(let everywhere also means that things break if you throw a try around something while debugging. I really think they both have their purposes.)

In my experience, if something is throwing, you already know where it's throwing from, so you wrap a single line in try-catch and that's trivial to change as needed.

In conclusion, I'll take safety over convenience.